### DISCUSSION TITLE
现在2.7版本还能玩吗?

### DISCUSSION THREAD

**Discussion by dxfhu2012** [Melledy/Grasscutter#1142] [Q&A] [answered]
刚开始使用这个，服务器已经搭建出来，显示在本地443，然后用fiddler加载了脚本，但是登录界面显示的还是官服的界面，而且无法登录。另外想问一下登录时候密码是什么呢？谢谢！

**Comment by zhaodice** [comment, marked as answer]
你没配好，我这边表示能玩，并准备实现抓螃蟹蝴蝶蜥蜴和鱼的功能

**Reply by dxfhu2012** [reply]
啊，大佬！我看你回复那个朋友的帖子了，想询问下我这个按照运行过程配置之后，也显示服务端运行了，但是我还是卡在登录页面不知道为啥，但是服务端好像不显示我登录成功

**Reply by zhaodice** [reply]
> 啊，大佬！我看你回复那个朋友的帖子了，想询问下我这个按照运行过程配置之后，也显示服务端运行了，但是我还是卡在登录页面不知道为啥，但是服务端好像不显示我登录成功

卡登录页面有3个可能:
1. 未能正确连接服务器(在客户机打开https://account.hoyoverse.com 可以验证，如果打开不是grasscutter那就是这条)
2. 未能正确安装证书(如果打开上述地址显示证书错误，那就是这条)
3. 安卓端不信任安装的证书，需要apk-mitm破解糜蛤油客户端，或者手机root直接安装为系统证书

**Reply by basaka2022** [reply]
> 你没配好,我这边表示能玩,并准备实现抓螃蟹蝴蝶蜥蜴和鱼的功能

大佬啥时候把掉肉的动物修一修

**Reply by zhaodice** [reply]
> > 你没配好,我这边表示能玩,并准备实现抓螃蟹蝴蝶蜥蜴和鱼的功能
> 
> 大佬啥时候把掉肉的动物修一修

我早就修了，你需要删除/data/drop.json 重启服务器进行更新

**Reply by basaka2022** [reply]
已经有了  感谢

**Comment by zhaodice** [comment]
当然如果你有自定义在drop.json，自己备份咯
